src/clj/game/cards/identities.clj

```diff
@@ -491,6 +491,18 @@
 (defcard "Chaos Theory: Wünderkind"
   {:static-abilities [(mu+ 1)]})
 
+(defcard "Chronos Protocol: Haas-Bioroid"
+  {:events [{:event :damage
+             :req (req (= (:damage-type context) :brain))
+             :msg (msg "remove all copies of " (enumerate-str (map :title (:cards-trashed context))) ", everywhere, from the game")
+             :effect (req (doseq [c (:cards-trashed context)]
+                            (let [all-candidates (filter #(and (not= (:zone %) [:rfg])
+                                                               (runner? %)
+                                                               (= (:title %) (:title c)))
+                                                         (get-all-cards state))]
+                              (doseq [candidate all-candidates]
+                                (move state :runner candidate :rfg)))))}]})
```

Comment:
This can actually be one `doseq`:

```clojure
(defcard "Chronos Protocol: Haas-Bioroid"
  {:events [{:event :damage
             :req (req (= (:damage-type context) :brain))
             :msg (msg "remove all copies of " (enumerate-str (map :title (:cards-trashed context))) ", everywhere, from the game")
             :effect (req (doseq [c (:cards-trashed context)
                                  :let [all-candidates (filter #(and (not (in-rfg? %))
                                                                     (runner? %)
                                                                     (= (:title %) (:title c)))
                                                               (get-all-cards state))]
                                  candidate all-candidates]
                            (move state :runner candidate :rfg)))}]})
```

The reason to do it this way is that it produces faster bytecode.

Reply:
neat